docs/upgrade.html

```diff
@@ -30,6 +30,12 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
             </li>
         </ul>
     </ul>
+    <ul>
+        <li>A number of deprecated configurations have been removed, with some configurations fully deleted and others having outdated enums removed.</li>
```

Comment:
we don't need this duplicate description, right?

Reply:
The previous description has no `configuration`.
I think the`remove configuration description` will be a lot of code. So I wrote another section.
I removed it now, PTAL.

Reply:
I feel you can add "configutation" to it :)

Reply:
please check this version